src/tools/sssctl/sssctl_config.c

```diff
@@ -92,16 +97,17 @@ errno_t sssctl_config_check(struct sss_cmdline *cmdline,
         goto done;
     }
 
-    if (config_path != NULL) {
+    if (config_path == NULL) {
+        config_path = SSSD_CONFIG_FILE;
+    }
+
+    if (config_snippet_path == NULL) {
         config_snippet_path = sssctl_config_snippet_path(tmp_ctx, config_path);
         if (config_snippet_path == NULL) {
             DEBUG(SSSDBG_CRIT_FAILURE, "Unable to create snippet path\n");
             ret = ENOMEM;
             goto done;
         }
-    } else {
-        config_path = SSSD_CONFIG_FILE;
-        config_snippet_path = CONFDB_DEFAULT_CONFIG_DIR;
```

Comment:
If CONFDB_DEFAULT_CONFIG_DIR can be removed now or if it is still used somewhere?

Reply:
The CONFDB_DEFAULT_CONFIG_DIR is still used elsewhere.